CesiumGeospatial/include/CesiumGeospatial/EarthGravitationalModel96Grid.h

```diff
@@ -0,0 +1,60 @@
+#pragma once
+
+#include "Library.h"
+
+#include <CesiumGeospatial/Cartographic.h>
+
+#include <gsl/span>
+
+#include <cstdint>
+#include <optional>
+#include <string>
+#include <vector>
+
+namespace CesiumGeospatial {
+
+/**
+ * @brief Allows elevation data in the format provided by the Earth
+ * Gravitational Model 1996 to be loaded and queried.
```

Comment:
Active voice would be better:

```suggestion
 * @brief Loads and queries heights from the Earth Gravity Model 1996 (EGM96).
```

Just call me the MS Word grammar checker. 😎 

Also elevation -> height. The word elevation tends to imply a height above sea level, so we avoid it.